devsetup/edpm/edpm-play.yaml

```diff
@@ -24,6 +24,9 @@ spec:
     - name: Deploy EDPM facts playbook
       ansible.builtin.import_playbook: deploy-edpm-facts.yml
 
+    - name: Deploy EDPM growvols playbook
```

Comment:
> in #154 I set the growvols as a firstboot-command - not sure if I'm right calling it that early then... ?

That is reasonable for now, but growpart, and xfs_growfs are the grow commands required for centos-9-stream-base.qcow2. growvols is an entirely separate script used by the yet-to-be-released edpm-hardened-uefi.qcow2. The intention is for this change to land, but the playbook will be a noop until the image switches over.

[See here](https://github.com/steveb/edpm-image-builder) if you are curious about this new image, I'll propose the new repo to openstack-k8s-operators as soon as dependencies are ready.

We can discuss the options later for how to transition to edpm-hardened-uefi.qcow2, hopefully it can be a single switchover commit to gen-edpm-compute-node.sh which can also remove other grow calls.

Reply:
https://github.com/openstack-k8s-operators/edpm-image-builder/ has been imported now